src/sif/autocost.cc

```diff
@@ -137,7 +137,7 @@ class AutoCost : public DynamicCost {
    * Construct auto costing. Pass in cost type and costing_options using protocol buffer(pbf).
    * @param  costing_options pbf with request costing_options.
    */
-  AutoCost(const CostingOptions& costing_options, uint32_t access_mask = kAutoAccess);
+  AutoCost(const CostingOptions& costing_options, uint32_t access_mask = (kAutoAccess | kHOVAccess));
```

Comment:
dont forget to completely delete HOV costing below and add the code to worker.cc to see that costing string and make the changes to the request to turn on HOV2 or somethign for the auto costing

Reply:
@kevinkreiser I'm fuzzy on what exactly it means to "delete the HOV costing".  Like, literally?  Or leave it as a shell class that just defaults to AutoCost with HOV2?

Reply:
i meant literally :smile: 

basically 2 steps:

1. delete HOV costing from autocost.cc
2. in worker.cc if someone asks for a `costing` in the request as `hov` then we set `costing` to `auto` (in the actual request json) and we rewrite `costing_options.hov` to `costing_options.auto` (again in the actual request json). and then we add to `costing_options.auto` the `hov2` flag. then we let the request parsing go on as normal.

there are a number of other deprecated features that work this way. have a look over in worker.cc and search for "deprecate" and you'll see a couple places where this happens. not specifically for costing but for other parameters we've sunset

Reply:
There’s even the deprecated “auto_shorter” costing in there as well IIRC